File: trunk/v1.0.0/src/com/youthor/bsp/core/abdf/common/model/validation/Validation.java
package com.youthor.bsp.core.abdf.common.model.validation;

import java.util.ArrayList;
import java.util.List;

public class Validation {
    private String name;
    private String param;
    private String displayError;
    private ValidationField validationField;
    private String fire;
    private String fireValue;
    private boolean atServer;
	public boolean isAtServer() {
		return atServer;
	}
	public void setAtServer(boolean atServer) {
		this.atServer = atServer;
	}
	public String getFire() {
		return fire;
	}
	public void setFire(String fire) {
		this.fire = fire;
	}
	public String getFireValue() {
		return fireValue;
	}
	public void setFireValue(String fireValue) {
		this.fireValue = fireValue;
	}
	public ValidationField getValidationField() {
		return validationField;
	}
	public void setValidationField(ValidationField validationField) {
		this.validationField = validationField;
	}
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public String getParam() {
		return param;
	}
	public void setParam(String param) {
		this.param = param;
	}
	public String getDisplayError() {
		return displayError;
	}
	public void setDisplayError(String displayError) {
		this.displayError = displayError;
	}
	
}

File: trunk/v1.0.0/WebContent/bsp/abdf/js/validation/Validation.js
function Validation() {
    this.name = null;
    this.params = null;
    this.displayError = null;
    this.atClient=true;
    this.fire=null;
    this.fireValue=null;
     this.form = null;
    
    this.getFireValue = function() { return this.fireValue; }
    this.setFireValue = function(p0) { this.fireValue = p0; }
    
    this.getFire = function() { return this.fire; }
    this.setFire = function(p0) { this.fire = p0; }
    
    this.getAtClient = function() { return this.atClient; }
    this.setAtClient = function(p0) { this.atClient = p0; }

    this.getName = function() { return this.name; }
    this.setName = function(p0) { this.name = p0; }

    this.getParams = function() { return this.params; }
    this.setParams = function(p0) { this.params = p0; }

    this.getDisplayError = function() { return this.displayError; }
    this.setDisplayError = function(p0) { this.displayError = p0; }
    
     this.getForm = function() { return this.form; }
    this.setForm = function(p0) { this.form = p0; }
}